README.md

```diff
@@ -1,77 +1,90 @@
 # Enterovirus D68 Nextstrain Analysis
 Performs a full Nextstrain analysis on Enterovirus D68 - currently a >=700bp VP1 run and a >=6000bp full-genome run. More may be added in future. 
 
-This repository will also eventually include code to update publicly available sequences by comparing new ViPR downloads against what's already part of the most recent run, but this isn't implemented yet. However, instructions below include this already (even though it's not yet implemented).
+**_Important note:_** This repository is currently in the process of being adapted to a new method of fetching new sequences, after the sunsetting (discontinuation) of ViPR. The intermediate solution is _not ideal_ but is temporary while we transition. Instructions below will adapt to reflect the current best way to run builds! 
+
+**Thank you for your patience and understanding during this transition!**
+
+Currently, the code will construct a local 'database' of the sequences you already have downloaded from NCBI. When you fetch new metadata from NCBI Virus, it will compare what's new against what you already have, and only download new sequences.
 
 ### Data
 Note that the data used for these runs is *not* part of this repository. You can download publicly available data using the instructions below to add to your own run, and add any of your own data, too (see notes below on this).
 
-## Quickstart 
-### Setup
+## Getting Started 
+
+To download files from NCBI you will need to provide an email address. This should be put in a `.env` file in the base directory, in the format `EMAIL=user@email.com` (add the `.env` file to your `.gitignore` to stop it being pushed publicly). To use this method of accessing the email, you will need to install the `dotenv` package - you can do this via PyPI with `pip install python-dotenv` (or via conda/micromamba with `conda install -c bioconda python-dotenv`)
```

Comment:
email.com is an actual real domain -- consider changing this like so; example.com is also an actual real domain but it is reserved for exactly this type of usage:

```suggestion
To download files from NCBI you will need to provide an email address. This should be put in a `.env` file in the base directory, in the format `EMAIL=user@example.com` (add the `.env` file to your `.gitignore` to stop it being pushed publicly). To use this method of accessing the email, you will need to install the `dotenv` package - you can do this via PyPI with `pip install python-dotenv` (or via conda/micromamba with `conda install -c bioconda python-dotenv`)
```